### ISSUE TITLE
关于bce和lovasz hinge的加权问题

### ISSUE THREAD

**Issue by yu-Mas** [PaddlePaddle/PaddleSeg#704]
你好，很高兴能看到这么详细的中文文档，对我帮助颇大，谢谢你们！

https://github.com/PaddlePaddle/PaddleSeg/blob/release/v0.8.0/docs/lovasz_loss.md
我在阅读上述lovasz文档时有个困扰，当bce和lovasz hinge结合时，bce内部使用sigmoid后再计算，lovasz hinge则直接使用logits计算。那么使用metric（譬如dice或iou）来评价时，假如阈值固定，计算metric的对象是sigmoid（logits）还是logits？

不知道表述的请不清晰，期待你们的回复。
谢谢

**Comment by LutaoChu** [comment]
很高兴你对PaddleSeg文档的认可！
我理解计算metric的对象是predict的结果。对于lovasz hinge和bce和dice loss，会经过sigmoid，目的是将单通道的logit变换为两通道，最终的对象为argmax(sigmoid(logits))。对于其他loss为argmax(logits)
